UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -819,17 +834,315 @@ public IEnumerator ProcessDNAPayload(DNAMutationData InDNAMutationData)
 
 							if (Payload.SpeciesMutateTo != null && Payload.MutateToBodyPart != null)
 							{
-								Mutation.ChangeToSpecies(Payload.SpeciesMutateTo,
-									Payload.MutateToBodyPart);
+								Mutation.ChangeToSpecies(Payload.MutateToBodyPart);
 							}
 						}
 					}
 				}
 			}
 		}
 
-		#endregion
+		private void UploadDnaDataToBodyParts(CharacterSheet characterSheet, List<DNAMutationData> InDNAMutationDatas)
```

Comment:
would be better if this was a body part functionality, if there was a core element of a ling

Reply:
For what of functionality inside that?

Reply:
so functionality would be on the Brain of the ling, that would be unique to the ling

Reply:
Still don`t get it....